### DISCUSSION TITLE
Side slip angle issue

### DISCUSSION THREAD

**Discussion by JBRDLR** [DLR-VSDC/IEEE-MVC-2023#30] [Q&A]
### Discussed in https://github.com/DLR-VSDC/IEEE-MVC-2023/discussions/23

<div type='discussions-op-text'>

<sup>Originally posted by **woojmn** February  2, 2023</sup>
Hello,

I have a question about the side slip angle.

![error_picture](https://user-images.githubusercontent.com/95162423/216268807-cd9371b4-0440-421e-a335-1bd138fa4828.PNG)

As you can see the picture, in the cycle #4, the steer and yaw rate are almost zero but the side slip angle is large value. 

I think it is weird, so I wonder your idea for this issue.</div>

**Comment by rpintodecastro** [comment]
Potential reasons for this issue
i) numerical issues when velocity is close to zero
ii) since this is an angle, +pi and -pi represent the same information; so you might see jumps from +pi and -pi (and vice-versa),

**Comment by tobolar** [comment]
> Potential reasons for this issue

This seems to very likely be the issue.

@woojmn: could you provide or check `longitudinalVelocity` as well?
